bindings/java/c/evmc-vm.c

```diff
@@ -138,6 +138,18 @@ JNIEXPORT jint JNICALL Java_org_ethereum_evmc_EvmcVm_set_1option(JNIEnv* jenv,
     return (jint)option_result;
 }
 
+JNIEXPORT jlong JNICALL Java_org_ethereum_evmc_EvmcVm_address(JNIEnv* jenv,
+                                                              jclass jcls,
+                                                              jobject buf)
+{
+    (void)jcls;
+#ifdef DEBUG
+    printf("Entering address\n");
+#endif
```

Comment:
Debug printouts should probably be removed

Reply:
I put a `#ifdef DEBUG` as a precompile directive. Is that not proper?

Reply:
removed